frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/setup.tsx

```diff
@@ -41,22 +46,12 @@ export const setupEmbedding = async ({
   return { ...returnedValue, history: checkNotNull(returnedValue.history) };
 };
 
-export const goToStaticEmbeddingSettings = async () => {
```

Comment:
Remove these 2 functions because there are variants when on EE, the label would be something else, so I put that in the test instead.